ipatests/test_integration/test_subids.py

```diff
@@ -268,3 +269,20 @@ def test_sudid_match_shows_uid(self):
                                           f"--subuid={subuid}"])
         owner = self._parse_result(result)["owner"]
         assert owner == uid
+
+    def test_nsswitch_is_updated_with_subid_entry(self):
+        """
+        This test case checks that when ipa-client-install
+        is installed with --subid option, the nsswitch.conf
+        file is modified with the entry 'subid: sss'
+        """
+        tasks.uninstall_client(self.clients[0])
+        tasks.install_client(
+            self.master, self.clients[0], extra_args=['--subid'],
+            nameservers=None
+        )
+        cmd = self.clients[0].run_command(
+            ["grep", "^subid", "/etc/nsswitch.conf"]
+        )
+        subid = cmd.stdout_text.split()
```

Comment:
Please use `split` with `maxsplit=1`. And it's safer to strip whitespaces around the value.

Reply:
@stanislavlevin added maxsplit=1